go.mod

```diff
@@ -4,7 +4,11 @@ go 1.17
 
 require (
 	github.com/anmitsu/go-shlex v0.0.0-20200514113438-38f4b401e2be
+	github.com/charmbracelet/x/exp/term v0.0.0-20240117030132-5a84c80527c7
+	github.com/creack/pty v1.1.21
+	github.com/u-root/u-root v0.11.0
```

Comment:
Note that this package doesn't support `solaris`. We could switch to [`pkg/term`](https://github.com/pkg/term/tree/master/termios) or simply drop support for `solaris` for now